scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/XMLTags.java

```diff
@@ -135,6 +135,14 @@ public String getXMLName() {
         return this.xmlName;
     }
 
+    public String getOpenTagPattern() {
+        return String.format("<[ ]*%s[ ]*>", this.xmlName);
+    }
+
+    public String getCloseTagPattern() {
+        return String.format("<[ ]*/[ ]*%s[ ]*>", this.xmlName);
```

Comment:
can u extract a final constant with a readable name out of "<[ ]*/[ ]*%s[ ]*>" ?